BuildPropertiesPlugin/sample/app/build.gradle

```diff
@@ -1,15 +1,11 @@
 buildscript {
-  repositories {
-    maven { url rootProject.file('../.gradle/repo') }
-  }
   dependencies {
     classpath 'com.android.tools.build:gradle:2.1.2'
-    classpath "${pluginGroup}:${pluginId}:${pluginVersion}"
   }
 }
 
 apply plugin: 'com.android.application'
-apply plugin: 'build-properties'
+apply plugin: com.novoda.buildproperties.BuildPropertiesPlugin
```

Comment:
This is not the standard way of applying the plugin (note the lack of ''). The sample is now compiling against the plugin sources directly, therefore we need to use the class name instead of the string identifier.